### PR TITLE
Add MIN_SAFE_INTEGER

### PR DESCRIPTION
**Pull request by onishiweb**

### CODE REVIEW

File: polyfills/Number/MIN_SAFE_INTEGER/config.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		"firefox": "<31",
+		"firefox_mob":"<32",
+		"chrome":"<34",
+		"android": "<5",
+		"safari": "<9",
+		"ios_saf": "<9",
+		"opera": "*",
+		"op_mini": "*",
+		"op_mob": "*",
+		"bb": "*"
```
Comment (ruiaraujo) [inline polyfills/Number/MIN_SAFE_INTEGER/config.json:17]: Missing samsung browser here.

Comment (JakeChampion) [reply polyfills/Number/MIN_SAFE_INTEGER/config.json:17]: We don't know the support for Samsung browser -- @AdaRoseEdwards could you help here?

Comment (JakeChampion) [reply polyfills/Number/MIN_SAFE_INTEGER/config.json:17]: Not needed, see -- https://github.com/Financial-Times/polyfill-service/pull/885#discussion_r79589886

Comment (onishiweb) [reply polyfills/Number/MIN_SAFE_INTEGER/config.json:17]: 👍

**Review by rowanmanning — APPROVED** [review]
Looks good 👍

**Pull request merged**